docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+
+
+Step 2: (Optional) Boot the servers in single-user mode
+=======================================================
+If you do not have the original password for the shell admin account on the
+*Application* and *Monitor Servers*, you'll need to reset the password on each
+server by booting in single user mode. In order to do so, you'll need physical
+access to the server, a keyboard, and a monitor. 
+
+First, connect a monitor and keyboard to the *Monitor Server*. Then reboot the server.
+When the GRUB menu appears, make sure the **Ubuntu** entry is highlighted, and 
+press **e** to edit boot options.
+In the boot options for Ubuntu, find the line that starts with ``linux`` and ends
+with ``ro``. Add ``single`` after ``ro``, separated by a space, and press
+**F10** to boot in single user mode.
+
+Reset the SecureDrop admin user's password
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Once the root prompt appears, you'll need to reset the password for the
+SecureDrop admin user. By default this user is named `sdadmin` and has UID 1000.
+However it may have been named differently during the installation of your 
+instance. You can use the command ``getent passwd 1000`` to check the username
+corresponding to UID 1000. Once you have the correct username, reset its password
+using the `passwd` command, for example:
+
+.. code:: sh 
+
+ passwd sdadmin
+
+.. important::
+ Make sure to select a strong password, and record it in the *Admin Workstation's*
+ KeePassX database.
+
+Finally, reboot the *Monitor Server* and verify that you can log in at the console
+using the new password.
+
+Repeat the process for the *Application Server*. Use the same username and 
+password as for the *Monitor Server* - this is required in order for the 
+``./securedrop-admin install`` command to work correctly.
+
+Step 3: Set up Admin Workstation access
+=======================================
+Next, you'll configure the servers to allow temporary SSH access from the new *Admin
+Workstation*.
+
+First, start the new *Admin Workstation* with persistence enabled and an administration
+password set.
+
+Next, connect the new *Admin Workstation* to the *Hardware Firewall* via the
+appropriate Ethernet port, and set up its static IP address. For more information
+on how to do so, see :ref:`this section in the firewall setup documentation
+<assign_static_ip_to_workstation>`. If you do not know the correct static IP
+address for the *Admin Workstation*, log in to the *Hardware Firewall's* admin
+interface and check the settings under ``Aliases``.
+
+Next, determine whether your instance was set up to allow adminstrative access
+via SSH over Tor, or via SSH over LAN. If you don't know which option was originally
+chosen, you can check as follows:
+
+ #. Log in to the *Application Server* via the console using the adminstration username
+    and password.
+ #. Check to see if a file ``/var/lib/services/tor/ssh/hostname`` exists. If it
```

Comment:
- Should be `/var/lib/tor/services/ssh/hostname`, not  `/var/lib/services/tor/ssh/hostname`
- Since it requires `sudo`, maybe provide command to check? `sudo cat /var/lib/tor/services/ssh/hostname` or similar

Reply:
Fixed.